test/documentation/DoxygenLayout.xml

```diff
@@ -23,6 +23,7 @@
         </tab>
         <tab type="usergroup" visible="yes" title="How-To" intro="">
             <tab type="user" visible="yes" title="Subcommand argument parsing" url="\ref subcommand_arg_parse" intro=""/>
+            <tab type="user" visible="yes" title="Use cereal" url="\ref howto_use_cereal" intro=""/>
```

Comment:
```suggestion
            <tab type="user" visible="yes" title="Serialisize SeqAn data structures" url="\ref howto_use_cereal" intro=""/>
```
or `Serialise` ? :see_no_evil: I don't remember if `s` or `z`

Reply:
I would use Serialize? That is how we call the functions in the data structures.